packages/salesforcedx-vscode-lwc/src/index.ts

```diff
@@ -7,32 +7,38 @@
 
 import * as lwcLanguageServer from 'lwc-language-server';
 import * as path from 'path';
-import {
-  ConfigurationTarget,
-  ExtensionContext,
-  workspace,
-  WorkspaceConfiguration
-} from 'vscode';
+import * as vscode from 'vscode';
 import {
   LanguageClient,
   LanguageClientOptions,
   ServerOptions,
   TransportKind
 } from 'vscode-languageclient';
+import { forceLightningLwcCreate } from './commands/forceLightningLwcCreate';
 import { ESLINT_NODEPATH_CONFIG, LWC_EXTENSION_NAME } from './constants';
 
-export async function activate(context: ExtensionContext) {
+function registerCommands(): vscode.Disposable {
+  // Customer-facing commands
+  const forceLightningLwcCreateCmd = vscode.commands.registerCommand(
+    'sfdx.force.lightning.lwc.create',
+    forceLightningLwcCreate
+  );
+
+  return vscode.Disposable.from(forceLightningLwcCreateCmd);
+}
+
+export async function activate(context: vscode.ExtensionContext) {
```

Comment:
🐛 - I have to be careful while activating this command. It's actually possible for an extension to be installed without any of its extension-dependencies being installed (e.g, the user doesn't say install all dependencies, the user forcibly deletes the dependencies after, etc).

I need to have a check here for to ensure we don't activate if salesforcedx.salesforcedx-vscode-core exists and it has the api exported.

This is a bit more crucial because of the way the `forceLightningLwcCreate` gets its dependencies since the series of const on the top of that file.

**In the future, `forceLightningLwcCreate` is likely to move back into salesforcedx-vscode-core with the other create commands but for the time being, it's important to put it under here*.